src/global.css

```diff
@@ -11,6 +11,20 @@
   --fc-text-primary: #393939;
   --fc-text-secondary: #111;
   --fc-content-passive: #d4d4d4;
+
+  /* -----Headlines----- */
+  --h1-fs: 2.93rem;
+  --h1-fw: 500;
+  --h2-fs: 2rem;
+  --h2-fw: 500;
+  --h3-fs: 1.25;
+  --h3-fw: 600;
+  --h4-fs: 0.93rem;
+  --h4-fw: 600;
+  --h5-fs: 0.8rem;
+  --h5-fw: 400;
+  --h6-fs: 0.8rem;
+  --h6-fw: 1rem;
```

Comment:
Everything that is repeating itself could be one variable - you also already declared variables for the `font-weight` in line 5, 6 and 8 which you should use in your `typography.css`. 
Also, I wouldn't create variables for the sizes as they are pretty unique to the headings and you can't use them anywhere else because of the name. 
```suggestion
```

Reply:
Thank you!